src/python/pants/backend/docker/subsystems/docker_options.py

```diff
@@ -121,6 +122,14 @@ class DockerOptions(Subsystem):
             "build for at execution time."
         ),
     )
+    build_verbose = BoolOption(
+        "--build-verbose",
+        default=False,
+        help=(
+            "Prints build output to console when `--build-verbose` is true, otherwise only the "
```

Comment:
Usually we phrase the help more succinctly, e.g., "Whether to log the Docker output to the console. If false, only the image ID is logged."